sdk/storage/storage-blob/review/storage-blob.api.md

```diff
@@ -1325,6 +1356,8 @@ export interface ContainerGetPropertiesHeaders {
     blobPublicAccess?: PublicAccessType;
     clientRequestId?: string;
     date?: Date;
+    defaultEncryptionScope?: string;
+    denyEncryptionScopeOverride?: boolean;
```

Comment:
should we group the two into an object so they can be round-tripped?

Reply:
like, use the `ContainerEncryptionScope` type discussed above https://github.com/Azure/azure-sdk-for-js/pull/7068#discussion_r376050805.

Reply:
> like, use the `ContainerEncryptionScope` type discussed above [#7068 (comment)](https://github.com/Azure/azure-sdk-for-js/pull/7068#discussion_r376050805).

But I find that most of the response headers are flat. I was trying hard to find a counterexample   .

Reply:
It's find if we don't do this. I think .NET and Java also have loose parameters for these.

Reply:
I was thinking passing in and returning the same type in convenience layer.  I didn't meant to change the generated code.